fuel/build.gradle

```diff
@@ -15,6 +15,7 @@ targetCompatibility = JavaVersion.VERSION_1_6
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     compile "com.github.kittinunf.result:result:$resultVersion"
+    compile 'org.json:json:20170516'
```

Comment:
Hmm, do we use this somewhere?

Reply:
Yep, we need it for tests to manipulate and create a json, dunno why I was not able to get any other access to JSON

Reply:
For tests? Then, perhaps using `testCompile` is better?

Reply:
I need that to create a JSON object to test sending that as body

```
val json = JSONObject()
                         json.put("id", this.value)
                         json.toString()
```

Reply:
Yes, here right?

```
  is GetParamsTest -> {
    val json = JSONObject()
    json.put("id", this.value)
    json.toString()
```

But it is only in the test right? So you can technically just do ` testCompile 'org.json:json:20170516'` so that it doesn't forcefully push dependencies to the users.

Reply:
Yep, fixed switching it to `testCompile`